src/core/internal/elf/dl.d

```diff
@@ -76,6 +81,7 @@ struct SharedObject
     {
         version (linux)       enum IterateManually = true;
         else version (NetBSD) enum IterateManually = true;
+        else version (OpenBSD) enum IterateManually = true;
         else                  enum IterateManually = false;
```

Comment:
Please keep these aligned

Reply:
Done in latest (forced update, to keep commit squashed)